src/libNetwork/P2PComm.cpp

```diff
@@ -1065,15 +1112,11 @@ void P2PComm ::EventCbClientSeed([[gnu::unused]] struct bufferevent* bev,
     if (destBytes != NULL) {
       if (bufferevent_write(bev, &(destBytes->at(0)), destBytes->size()) < 0) {
         LOG_GENERAL(WARNING, "Error: P2PSeed bufferevent_write failed !!!");
-        return;
       }
+      struct timeval tv = {SEED_SYNC_LARGE_PULL_INTERVAL, 0};
+      bufferevent_set_timeouts(bev, &tv, NULL);
     }
-  }
-  if (events & (BEV_EVENT_EOF | BEV_EVENT_ERROR)) {
-    bufferevent_set_timeouts(bev, NULL, NULL);
-    CloseAndFreeBevP2PSeedConnClient(bev, ctx);
-  }
-  if (events & BEV_EVENT_TIMEOUT) {
+  } else if (events & (BEV_EVENT_EOF | BEV_EVENT_ERROR | BEV_EVENT_TIMEOUT)) {
```

Comment:
No need to clear the timeout anymore?

Reply:
No.Actually, we are freeing bufferevent , so timeout would be automatically cleared.